zkevm-circuits/src/evm_circuit/op_execution/utils.rs

```diff
@@ -10,6 +10,11 @@ pub(crate) mod common_cases;
 pub(crate) mod constraint_builder;
 pub(crate) mod math_gadgets;
 
+/// The maximum number of bytes that a field element
+/// can be broken down into without causing the value it
+/// represents to overflow a single field element.
+pub const MAX_BYTES_FIELD: usize = 31;
```

Comment:
It'll be better if we move this const to `src/evm_circuit/param.rs`.

Reply:
Done! There's some other constants I introduced in my other PRs that I will also move there after this PR is merged.